config/initializers/sentry.rb

```diff
@@ -13,6 +13,17 @@ def filter_record_not_unique_exception_messages!(event, hint)
   end
 end
 
+def ignore_sidekiq_errors_if_retry(event)
+  job = event&.transaction&.context&.job
+  if event&.exception.is_a?(Faraday::Error) &&
+       (
+         job.is_a?(ActiveStorage::AnalyzeJob) ||
+           job.is_a?(ActiveStorage::PurgeJob)
+       ) && job.retry?
+    true
+  end
```

Comment:
This would work, but whenever you've got something like this, you can remove the `if` statement. For example:

```rb
if <expression>
  return true
end
```

Can just become:

```
return <expression>
```

Since the expression itself will evaluate to true or false.